frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -69,6 +74,10 @@ function VariableItem({
 		getOptions();
 	}, [getOptions]);
 
+	const onRefreshHandler = (): void => {
+		updateTimeInterval(getDefaultOption(location.pathname));
```

Comment:
update time interval take selected interval and min max time in second

Reply:
@palashgdev 
The problem was, when we select any variables from dropdown, we needs to auto refresh manually.
So, I added refresh call on change of dropdown value. So no needs to do extrac CTA.